poznaj/templates/base.html

```diff
@@ -5,6 +5,53 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
   <title>{% block title %}{% endblock %}</title>
+
+
+  <style>
+      body {
```

Comment:
Can you add those as additional CSS file instead of having them here

Reply:
👍 ok